tcms/testruns/tests/test_views.py

```diff
@@ -555,6 +555,17 @@ def setUpTestData(cls):
         cls.intern = UserFactory(username='intern',
                                  email='intern@example.com')
 
+    def test_should_be_able_to_open_testrun_edit_page(self):
+        response = self.client.get(self.edit_url)
+
+        self.assertEqual(HTTPStatus.OK, response.status_code)
+        self.assertNotContains(
+            response,
+            '<input type="text" id="id_summary" name="summary" value="%s" class="form-control"'
+            ' required="">' % self.test_run.summary,
+            html=True
+        )
```

Comment:
This is a permissions related test, see `bugs/tests/test_permissions.py`. It should be modelled in the new syntax which is a separate change, not related to what we do here.